src/backend/columnar/columnar_reader.c

```diff
@@ -415,12 +415,12 @@ ReadChunkGroupNextRow(ChunkGroupReadState *chunkGroupReadState, Datum *columnVal
 	 */
 	memset(columnNulls, true, sizeof(bool) * chunkGroupReadState->columnCount);
 
-	Var *projectedColumn = NULL;
-	foreach_ptr(projectedColumn, chunkGroupReadState->projectedColumnList)
+	const ChunkData *chunkGroupData = chunkGroupReadState->chunkGroupData;
+	const int rowIndex = chunkGroupReadState->currentRow;
```

Comment:
minor: maybe we can keep those two lines within the loop to make them closer where they are used

Reply:
Done